lib/cypress/cql_bundle_importer.rb

```diff
@@ -103,18 +103,19 @@ def self.unpack_and_store_qdm_patients(zip, type, bundle)
       puts "\rLoading: Patients Complete          "
     end
 
+    # TODO: This will need to be updated for 2018.0.2 bundles that store relatedTo as an QDM::ID
```

Comment:
Does this `TODO` need to get resolved before the patch gets merged?

Reply:
Originally I was thinking that we would export the bundle using the new 0.8.4 model.  I now believe it is far less complex to export the bundle in the same 0.8.2 format and deal with the conversion on import.  That way both bundle will be imported the same way.  Fewer moving pieces.  I'll remove the TODO.